src/sphinx_book_theme/__init__.py

```diff
@@ -60,7 +60,7 @@ def add_metadata_to_page(app, pagename, templatename, context, doctree):
     context["translate"] = translation
 
     # If search text hasn't been manually specified, use a shorter one here
-    theme_options = app.config.html_theme_options or {}
+    theme_options = app.builder.theme_options or {}
```

Comment:
@choldgraf I think this has caused issues for the `pdflatex` builder (and will for other builders as well) as many builders won't have a `theme_options` object